scenes/game/game.gd

```diff
@@ -1099,6 +1101,18 @@ func _on_main_step_decision(event_data):
 				action_menu_choice_info["enable_check"].append(_allowed)
 				action_menu_choice_info["action_type"].append(action["action_type"])
 				action_menu_choice_info["actions_in_menu"].append(action)
+			elif action["action_type"] == Enums.GameAction_MainStepAttachedAction:
+				# List each unique attached action by effect id
+				var action_string = Strings.build_use_attached_action_string(action["effect_id"])
+				if action_string in action_menu_choice_info["strings"]:
+					# There's no other available marker that can be used to distinguish different
```

Comment:
This gets weird I guess if you have to fbk's each with fubuzilla attached.
Maybe it doesn't matter since it's still once per turn total?
Does make me wonder if we should server-side account for this and only have 1 fubuzilla action.